cmd/sovereignnode/chainSimulator/utils/utils.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	"math/big"
+
+	"github.com/multiversx/mx-chain-core-go/data/transaction"
+
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/configs"
+)
+
+func GenerateTransaction(
```

Comment:
Duplicated

Reply:
To reuse it from normal chain simulator tests, it needs a refactor for all the tests. So this is just for sovereign chain simulator a common function for all tests

Reply:
add comments.

Reply:
It is literally the same func, just checked it in compare tool. 
Delete one and keep only one

Reply:
This was moved in a common place